services/httpd/service.go

```diff
@@ -68,7 +69,8 @@ func NewService(c Config, hostname string, l *log.Logger, li logging.Interface)
 			li,
 			c.SharedSecret,
 		),
-		logger: l,
+		logger:           l,
+		httpServerLogger: li.NewStaticLevelLogger("[go-http]", log.LstdFlags, logging.ERROR),
```

Comment:
I think this should have the same name as the rest of the httpd service logging, `"[httpd]"`. Thoughts?